app/controllers/v0/financial_status_reports_controller.rb

```diff
@@ -105,8 +105,8 @@ def fsr_form
           :stocks_and_other_bonds,
           :real_estate_owned,
           :total_assets,
-          automobiles: %i[make model year resale_value],
-          other_assets: name_amount
+          { automobiles: %i[make model year resale_value],
+            other_assets: name_amount }
```

Comment:
I'm not going to complain about it now, but I think multi-line hashes shouldn't share the same line as the opening/closing braces.

I thought this was a default rule in rubocop. /shrug

Example:

```
{
  automobiles: %i[make model year resale_value],
  other_assets: name_amount
}
```

Reply:
I just looked it up. I guess the default styles doesn't care as long as it's "symmetrical". Well, community rules trump mine. 😄 

https://www.rubydoc.info/github/bbatsov/RuboCop/RuboCop/Cop/Layout/MultilineHashBraceLayout